### DISCUSSION TITLE
[QST]My laptop dont't have Nvidia GPU

### DISCUSSION THREAD

**Discussion by shiyi23** [NVIDIA/spark-rapids#5344] [General] [answered]
**What is your question?**
My laptop don‘t has Nvidia's independent graphics card, just has Intel's integrated graphics card, so can I run the RAPIDS Accelerator for Apache Spark？Or testing, debuging RAPIDS?

**Comment by jlowe** [comment, marked as answer]
Unfortunately, no.  The RAPIDS Accelerator is based on RAPIDS which requires an NVIDIA GPU with CUDA compute capability 6 or higher.  The hardware and software requirements for each release are documented on [the download page](https://nvidia.github.io/spark-rapids/docs/download.html).

**Comment by shiyi23** [comment]
> Unfortunately, no. The RAPIDS Accelerator is based on RAPIDS which requires an NVIDIA GPU with CUDA compute capability 6 or higher. The hardware and software requirements for each release are documented on [the download page](https://nvidia.github.io/spark-rapids/docs/download.html).

thanks for your answer~
